packages/blade/src/components/Scale/Scale.web.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { BaseMotionEnhancerBox } from '~components/BaseMotion';
+import type { MotionVariantsType } from '~components/BaseMotion';
+import { makeSecondsDuration } from '~utils/makeSecondsDuration';
+import { cssBezierToMotionFn } from '~utils/cssBezierToMotionFn';
+import { castWebType, useTheme } from '~utils';
+import type { ScaleProps } from './types';
+
+export const Scale = ({
+  children,
+  isHighlighted,
```

Comment:
what does isHighlighted prop do?

Reply:
in context of scale, `isHighlighted={true}` will scale the item up